### PR TITLE
scan asset results in manual add

### PR DESCRIPTION
**Pull request by piyalbasu**

### CODE REVIEW

**Comment by piyalbasu** [comment]
https://github.com/user-attachments/assets/261a3476-0804-44bb-a11a-4110c3d9b039

File: extension/src/popup/components/account/AssetDetail/styles.scss [hunk 1]
```diff
@@ -53,7 +53,7 @@
       font-weight: var(--font-weight-medium);
       justify-content: center;
       line-height: 150%;
-      margin: 2.5rem 0 0.5rem 0;
+      margin: -0.25rem 0 0.5rem 0;
```
Comment (piyalbasu) [inline extension/src/popup/components/account/AssetDetail/styles.scss:56]: small unrelated style fix

**Review by aristidesstaffieri — APPROVED** [review]
